kakarot_scripts/utils/kakarot.py

```diff
@@ -451,7 +514,9 @@ async def _wrapper(self, *args, **kwargs):
                 payload["data"] = list(payload["data"])
                 payload["origin"] = int(payload["from"], 16)
                 del payload["from"]
-                result = await kakarot_contract.functions["eth_call"].call(**payload)
+                result = await kakarot_contract.functions["eth_call"].call(
+                    **payload, block_number="pending"
```

Comment:
should pending not be the default argument?

Reply:
you mean keep latest as default?

Reply:
no i meant I thought the default argument (so block_number=None) resolves to "pending" but this actually might not be the case.

Reply:
actually this change looks useless since the default is

```
def _get_raw_block_identifier(
    block_hash: Optional[Union[Hash, Tag]] = None,
    block_number: Optional[Union[int, Tag]] = None,
) -> Union[dict, Hash, Tag, None]:
    if block_hash is not None and block_number is not None:
        raise ValueError(
            "Arguments block_hash and block_number are mutually exclusive."
        )

    if block_hash in ("latest", "pending") or block_number in ("latest", "pending"):
        return block_hash or block_number

    if block_hash is not None:
        return {"block_hash": _to_rpc_felt(block_hash)}

    if block_number is not None:
        return {"block_number": block_number}

    return "pending"
```